src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -32,7 +32,14 @@ The `LobHandler` (and respective API) has been deprecated for removal in Spring
 Respective option on `JdbcMessageStore` (and similar) have been deprecated as well.
 The byte array handling for serialized message is fully deferred to JDBC driver.
 
+The `LockRepository.delete()` method return the result of removing ownership of a distributed lock. And the `JdbcLockRegistry.JdbcLock.unlock()` method throws ConcurrentModificationException if the ownership of the lock is expired.
```

Comment:
DITTO.
Plus the `ConcurrentModificationException` is supposed to be a a code snippet like you do in other places.